src/logic/configure.ts

```diff
@@ -17,10 +18,48 @@ import {
 } from '../common/document';
 import { userError } from '../common/error';
 import { fetchProviderInfo } from '../common/http';
-import { readFile } from '../common/io';
+import { exists, readFile } from '../common/io';
 import { formatShellLog, LogCallback } from '../common/log';
 import { OutputStream } from '../common/output-stream';
+import { envVariable } from '../templates/env';
 
+export async function updateEnv(
+  provider: string,
+  securitySchemes: SecurityScheme[],
+  options?: { warnCb?: LogCallback }
+): Promise<void> {
+  let envContent = '';
+  //Get .env file
+  if (await exists('.env')) {
+    envContent = (await readFile('.env')).toString();
+  }
```

Comment:
Wouldn't be better to use `readFileQuiet` which returns `string | undefined`?

```suggestion
  let envContent: string | undefined;
  //Get .env file
  if (await exists('.env')) {
    envContent = await readFileQuiet('.env');
  }
```